JuliaInterface/gap/JuliaInterface.gd

```diff
@@ -203,6 +87,23 @@ DeclareGlobalFunction( "JuliaIncludeFile" );
 #!  and then load these files with <Ref Func="JuliaIncludeFile"/>.
 DeclareGlobalFunction( "JuliaImportPackage" );
 
+DeclareCategory( "IsJuliaModule", IsObject );
+BindGlobal( "TheFamilyOfJuliaModules", NewFamily( "TheFamilyOfJuliaModules" ) );
+BindGlobal( "TheTypeOfJuliaModules", NewType( TheFamilyOfJuliaModules, IsJuliaModule and IsAttributeStoringRep ) );
+
+#! @Description
+#!  TODO
+DeclareGlobalVariable( "Julia" );
+
+#! @Arguments module
+#! @Returns a list of strings
+#! @Description
+#!  Returns a list of names of components currently bound
+#!  in <A>module</A>. Please note that this does usually
+#!  not reflect all symbols bound to a module on the
+#!  &Julia; side.
```

Comment:
Perhaps be more explicit here.
I think the idea is that `module` shows what the GAP side currently knows about the Julia module,
and that there may be more symbols, even if the Julia module had been imported some time ago.
(In particular, the value returned by the function is not cached.)